app/model/Commands.scala

```diff
@@ -62,8 +62,20 @@ object Commands {
     hasAdjustments: Option[String], guaranteedInterview: Option[String], firstName: Option[String], lastName: Option[String],
     preferredName: Option[String], dateOfBirth: Option[String], cubiksUserId: Option[Int])
 
-  case class AdjustmentReport(userId: String, firstName: Option[String], lastName: Option[String], preferredName: Option[String],
-    email: Option[String], telephone: Option[String], adjustments: Option[String], gis: Option[String], adjustmentsConfirmed: Option[String])
+  final case class AdjustmentReport(
```

Comment:
We have created a new folder called model.report where you can put this class and related stuff in a separate file. Maybe you have to merge from master in to your branch to see what I mean